VAMobile/src/store/api/api.ts

```diff
@@ -210,6 +210,24 @@ const call = async function <T>(
     // No errors found, return the response
     return await response.json()
   } else {
+    const overrideErrors = _store?.getState().demo.overrideErrors
+    if (overrideErrors) {
+      _.forEach(overrideErrors, (error) => {
+        if (error.endpoint === endpoint) {
+          throw error.error
+        }
+        if (endpoint.includes(error.endpoint)) {
+          throw error.error
+        }
```

Comment:
When implementing this I only took into account get/fetch calls and not any other types of calls.

So those 2 use cases wouldn't have been caught for the includes.

I agree the 2nd if statement is redundant and would catch the first so I removed the first if statement.

The 2nd statement with the includes is needed though because of fetching secure messages, claims, appeals, etc... that have a specific ID. Instead of it having to match exactly for each individual one I opted to match the service itself. So all messages will fail or all claims will fail and the same for everything else that has an ID or path parameter. This is also why the last check is needed is because the ${folderID} conflicts